2007-07-31  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/unix/sysv/linux/posix_fallocate.c (posix_fallocate): Fix
	syscall arguments count.

--- libc/sysdeps/unix/sysv/linux/posix_fallocate.c.jj	2007-07-31 21:19:06.000000000 +0200
+++ libc/sysdeps/unix/sysv/linux/posix_fallocate.c	2007-07-31 21:19:06.000000000 +0200
@@ -39,7 +39,7 @@ posix_fallocate (int fd, __off_t offset,
 # endif
     {
       INTERNAL_SYSCALL_DECL (err);
-      int res = INTERNAL_SYSCALL (fallocate, err, 4, fd, 0,
+      int res = INTERNAL_SYSCALL (fallocate, err, 6, fd, 0,
 				  __LONG_LONG_PAIR (offset >> 31, offset),
 				  __LONG_LONG_PAIR (len >> 31, len));
 
